code/cookies.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel='icon' href='../favicon.ico' type='image/x-icon'>
+   <title>Cookie Policy &#8212; Textwork</title>
+   <link rel="stylesheet" type="text/css" href="_textwork-style.css">
+   <!---fonts--->
+</head>
+<body>
+    <section class="top-banner">
+        <div class="banner-elements">
+          <p><a href="mailto:hello@textwork.ch">hello@textwork.ch</a></p>
+          <p><a href="https://www.instagram.com/textwork_ch/" target="_blank" rel="noopener noreferrer"><img class="inst-sm" src="../img/glyph-logo_May2016.png" alt="textwork_ch"></a></p>
+          <p><a href="https://www.linkedin.com/in/jaksivio/" target="_blank" rel="noopener noreferrer"><img class="linkd-sm" src="../img/In-Black-28px-R.png" alt="jaksivio"></a></p>
+        </div>
+      </section>
+    
+    
+      <nav class="navcontainer">
+        <div class="navbar">
+            <ul>
+              <li><a href="./services.html">Services</a></li>
+              <li><a href="./about.html">About</a></li>
+              <!---<li><a href="./portfolio.html">Portfolio</a></li>-->
+              <li><a class="no-link" href="https://www.textwork.ch/"><img class="logo" src="../img/textwork_blogo.png" alt="Textwork-logo"></a></li>
+              <li><a href="./blog.html">Blog</a></li>
+              <li><a href="./contact.html">Contact</a></li>
+            </ul>
+          </div>
+      </nav>
+      <header class="bg-cookies">
+        <h1 class="headerh1">Cookie Policy</h1>
+      </header>
+
+    <div class="content">

```

Comment:
I really, really like that you've kept the lines short to make it easier for the reader. :)